modules/drivers/oracle/test/metabase/test/data/oracle.clj

```diff
@@ -265,3 +267,38 @@
                                     #_types          (into-array String ["TABLE"]))]
          ;; if the ResultSet returns anything we know the table is already loaded.
          (.next rset))))))
+
+(def ^:dynamic *override-describe-database-to-filter-by-db-name?*
```

Comment:
This code mostly copied from similar code I wrote for Redshift, fixes a broken test for Oracle that failed because Oracle "simulates" different test DBs by prefixing names which mean tests that look at all tables in a DB would fail since there would be extra stuff in there